Cloud Application Manager/Automating Deployments/template-box.md

```diff
@@ -1,68 +1,106 @@
 {{{ "title": "Template Boxes",
 "date": "09-01-2016",
 "author": "",
+"date": "12-28-2018",
+"author": "Julio Castanar",
+"keywords": ["cam","alm","boxes", "box", "templates", "box variables", "lifecycle editor", "binding", 
+        "CloudFormation Box Template", "Azure Box Template", "Terraform Box Template"],
 "attachments": [],
 "contentIsHTML": false
 }}}
-**In this article:**
 
+### Table of Contents
```

Comment:
Please, remove this file from your PR, since there is another PR on its way that updates it